model/src/gateway/event/mod.rs

```diff
@@ -349,3 +349,73 @@ impl Display for EventConversionError {
 }
 
 impl Error for EventConversionError {}
+
+#[cfg(test)]
+mod tests {
+    use super::{super::payload::incoming::*, shard::*, Event};
+    use static_assertions::const_assert;
+    use std::mem;
+
+    #[allow(dead_code)]
+    const EVENT_THRESHOLD: usize = 192;
```

Comment:
Why is this dead code exactly? Surely rustc knows this isn't?

Reply:
`const_assert` is internally super hacky, so no, it's considered dead code.